### ISSUE TITLE
Opening Mumble URL does not respect case

### ISSUE THREAD

**Issue by Kissaki** [mumble-voip/mumble#2585]
To reproduce:
* Create channel name `a`
* Create channel name `A`
* Copy URL to channel *A*
* Open URL

Expected: You join channel A

What happens: You join the first channel (*a*)

**Comment by ZeroAbility** [comment]
It looks like the PR for this was closed as stale. Is this a bug we intend to fix or has already been fixed?

**Comment by Krzmbrzl** [comment]
I don't think it has been fixed yet and yes, I believe we'll want to fix this. :thinking:

**Comment by vanarp123** [comment]
Hello, I was wondering if I could try fixing this ?

**Comment by davidebeatrici** [comment]
Sure!
